webpack.config.js

```diff
@@ -204,7 +204,7 @@ function generatePlugins({ isProduction, isPrerendering, scrivitoOrigin }) {
 }
 
 function devServerCspHeader() {
-  const directives = Object.assign({}, headersCsp);
+  const directives = deepClone(headersCsp);
```

Comment:
* [ ] We don't need to build our own `lodash.cloneDeep(headersCsp);`

Reply:
We currently only have `lodash-es` which AFAIK is not accessible in plain node - at least not in all node versions.

Since the method in question is a one-liner I think we can "roll our own".

Reply:
We're already using it in the generators, that's why I think it works.

Reply:
The generators have a dedicated `package.json` file (see https://github.com/Scrivito/scrivito_example_app_js/blob/master/generator-scrivito/package.json#L17 ).